rir/R/rir.R

```diff
@@ -32,28 +32,28 @@ pir.tests <- function() {
 pir.debugFlags <- function(ShowWarnings = FALSE,
                            DryRun = FALSE,
                            PreserveVersions = FALSE,
-                           DebugAllocator = FALSE,
                            PrintIntoFiles = FALSE,
                            PrintIntoStdout = FALSE,
                            PrintEarlyRir = FALSE,
                            PrintEarlyPir = FALSE,
                            PrintOptimizationPasses = FALSE,
                            PrintCSSA = FALSE,
-                           PrintLivenessIntervals = FALSE,
+                           PrintAllocator = FALSE,
                            PrintFinalPir = FALSE,
                            PrintFinalRir = FALSE) {
     # !!!  This list of arguments *must* be exactly equal to the   !!!
     # !!!    LIST_OF_PIR_DEBUGGING_FLAGS in compiler/debugging.h   !!!
     .Call("pir_debugFlags", ShowWarnings, DryRun, PreserveVersions,
-          DebugAllocator, PrintIntoFiles, PrintIntoStdout, PrintEarlyRir, PrintEarlyPir, PrintOptimizationPasses,
-          PrintCSSA, PrintLivenessIntervals, PrintFinalPir, PrintFinalRir,
+          PrintIntoFiles, PrintIntoStdout, PrintEarlyRir, PrintEarlyPir,
+          PrintOptimizationPasses, PrintCSSA, PrintAllocator, PrintFinalPir,
+          PrintFinalRir,
           # wants a dummy parameter at the end for technical reasons
           NULL)
 }
 
 # sets the default debug options for pir compiler
 pir.setDebugFlags <- function(debugFlags = pir.debugFlags()) {
-    .Call("pir_setDebugFlags", debugFlags)
+    invisible(.Call("pir_setDebugFlags", debugFlags))
```

Comment:
Why invisible is needed here?

Reply:
it's not needed, it just hides the returned NULL so that it isn't printed in the console